frontend/src/app/core/interceptors/error-handler.interseptor.ts

```diff
@@ -35,7 +37,7 @@ export class ErrorInterceptor implements HttpInterceptor {
         } else {
           this.notificationService.error('Error: unknown error');
         }
-
+        this.router.navigate(['']);
```

Comment:
it would be nice to check whether we are already here or not

Reply:
In case of network error on home page it will go in cycle

Reply:
There is no circle redirect. `router` doesn't allow redirecting on the same route.